src/schema/messaging/received.ts

```diff
@@ -0,0 +1,19 @@
+import { Type } from "@sinclair/typebox";
+
+import { defineEndpoint } from "@/generator-helpers.js";
+
+export default defineEndpoint({
+    source: "server",
+    target: "user",
+    description: "Notify the player a message has been received",
```

Comment:
Received as in, we send message out, and get notification that the other player received it? If there were multiple messages send out, which one?